data/json/furniture_and_terrain/terrain-roofs.json

```diff
@@ -304,12 +304,11 @@
     "looks_like": "t_linoleum_white",
     "color": "cyan",
     "move_cost": 2,
-    "trap": "tr_ledge",
-    "flags": [ "TRANSPARENT", "NO_FLOOR", "INDOORS" ],
+    "flags": [ "TRANSPARENT", "INDOORS" ],
     "bash": {
       "str_min": 3,
       "str_max": 6,
-      "sound": "glass braking!",
```

Comment:
This was correct actually, the glass was applying a braking effect to the vehicle, which is why it wasn't falling \s